services/brig/test/resources/turn/servers-v2.txt

```diff
@@ -1,2 +1 @@
-turn:localhost:3478
-turn:localhost:3479
+turn:localhost:3478
```

Comment:
unrelated change, undo?

Reply:
Very strange, no idea what happened here; great catch!

Reply:
Out of curiosity: when running brig's integration tests, one of the tests creates that 2nd entry... but it should always fallback to the original after, something funky happened there